### ISSUE TITLE
typo found

### ISSUE THREAD

**Issue by jpalma-espinosa** [susam/gitpr#4]
Hi,
while reading the pdf (0.5.0), at page 7, it says "The `git merge` command above[...]", however, there is no instruction stated on the use of that command. 
should it be?
```
git checkout master    
git pull upstream master 
git merge origin master
```

**Comment by susam** [comment]
Hi @jpalma-espinosa,

Thank you for reporting this issue. This issue was introduced in commit 6130ec83875189caaf6cc145ca3dd2c2b19c92d0. This issue has now been fixed in commit 90f74497efeff6c51442a400d50733460c2ef906.

I will close this issue after releasing the next version of the PDF.

**Comment by susam** [comment]
The fix for this issue is now available in [version 0.6.0](https://github.com/susam/gitpr/releases/download/0.6.0/gitpr.pdf).
